pkg/service/server.go

```diff
@@ -369,10 +372,28 @@ func (mgr *Manager) drainLoadbalancerTarget(event *LifecycleEvent) error {
 		return nil
 	}
 
+	// sleep for random jitter per goroutine
+	waitJitter(ThreadJitterRangeSeconds)
+
+	// add exclusion label
+	log.Debugf("excluding node %v from load balancers", node.Name)
+	err := labelNode(kubeClient, ctx.KubectlLocalPath, node.Name, ExcludeLabelKey, ExcludeLabelValue)
+	if err != nil {
+		return err
+	}
+
+	now := time.Now().UTC()
+	nodeCreationTime := node.CreationTimestamp.UTC()
+	nodeAge := int(now.Sub(nodeCreationTime).Minutes())
+	if nodeAge <= NodeAgeCacheTTL {
+		log.Infof("Node younger than 90m was terminated, flushing DescribeTargetHealth caches")
```

Comment:
could take advantage of formatting to include ttl const from above:
```suggestion
		log.Infof("Node younger than %v minutes was terminated, flushing DescribeTargetHealth caches", NodeAgeCacheTTL)
```